test/e2e/medusa_test.go

```diff
@@ -164,3 +169,28 @@ func verifyRestoreJobFinished(t *testing.T, ctx context.Context, f *framework.E2
 		return !restore.Status.FinishTime.IsZero()
 	}, polling.medusaRestoreDone.timeout, polling.medusaRestoreDone.interval, "restore didn't finish within timeout")
 }
+
+func checkMedusaStandaloneDeploymentExists(t *testing.T, ctx context.Context, dcKey framework.ClusterKey, f *framework.E2eFramework, kc *api.K8ssandraCluster) {
```

Comment:
Question:I'm not very good at reading these e2e tests, so maybe I'm missing something. But is there an e2e test here which actually exercises that a cluster can be backed up, destroyed, and then restored into another cluster which has a different name or different rack/DC names?

Reply:
More generally, we are very deficient on tests here. I can't find any test fixtures that reference the `medusabackupjobs.medusa.k8ssandra.io`, `medusabackupschedules.medusa.k8ssandra.io`, `medusatasks.medusa.k8ssandra.io`, `medusabackups.medusa.k8ssandra.io`, `medusarestorejobs.medusa.k8ssandra.io` etc. from a brief inspection.